src/js/view/floating/floating-controller.ts

```diff
@@ -237,13 +245,20 @@ export default class FloatingController {
         }
 
         this.disableFloatingUI();
-
-        // Perform resize and trigger "float" event responsively to prevent layout thrashing
-        this._model.trigger('forceResponsiveListener', {});
     }
     
     transitionFloating(isTransitionIn: boolean): void {
-        toggleClass(this._wrapperEl, 'jw-float-transition', isTransitionIn);
+        // Hiding the wrapper will impact player viewability momentarily, but reduce CLS
+        // We could hide the wrappers contents (controls, overlays) and reduce some CLS, but not as much
+        this._inTransition = isTransitionIn;
+        const wrapper = this._wrapperEl;
+        style(wrapper, {
+            display: isTransitionIn ? 'none' : null
+        });
```

Comment:
Good point. I think our best bet is to make line 251 - 253 gated by a config option. I can't come up with a (good) way to both keep this CLS score low and keep transition animations on mobile. As such I think it might be a good idea to make a config option for clients that prefer presentation to WV metrics.